apps/web-giddh/src/assets/locale/en.json

```diff
@@ -370,5 +370,7 @@
     "igst": "IGST",
     "cgst": "CGST",
     "sgst": "SGST",
-    "particular": "Particular"
+    "particular": "Particular",
```

Comment:
These locales are module specific, should be in module folder

Reply:
it's used in TB daybook export import that's why added in parent